<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from themewagon.github.io/elen/contact.html by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 08 Nov 2024 06:32:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <title>Snijesh-PortFolio</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    
    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,700" rel="stylesheet">

    <link rel="stylesheet" href="css/open-iconic-bootstrap.min.css">
    <link rel="stylesheet" href="css/animate.css">
    
    <link rel="stylesheet" href="css/owl.carousel.min.css">
    <link rel="stylesheet" href="css/owl.theme.default.min.css">
    <link rel="stylesheet" href="css/magnific-popup.css">

    <link rel="stylesheet" href="css/aos.css">

    <link rel="stylesheet" href="css/ionicons.min.css">

    <link rel="stylesheet" href="css/bootstrap-datepicker.css">
    <link rel="stylesheet" href="css/jquery.timepicker.css">

    
    <link rel="stylesheet" href="css/flaticon.css">
    <link rel="stylesheet" href="css/icomoon.css">
    <link rel="stylesheet" href="css/style.css">
  </head>
  <body>

	<div id="colorlib-page">
		<a href="#" class="js-colorlib-nav-toggle colorlib-nav-toggle"><i></i></a>
		<aside id="colorlib-aside" role="complementary" class="js-fullheight text-center">
			<h1 id="colorlib-logo"><a href="index.html">SNIJESH V P</a></h1>
			<nav id="colorlib-main-menu" role="navigation">
				<ul>
					<li><a href="index.html">Home</a></li>
					<li><a href="photography.html">Works</a></li>
					<li><a href="travel.html">Travel</a></li>
					<li><a href="fashion.html">Fashion</a></li>
					<li><a href="about.html">About</a></li>
					<li class="colorlib-active"><a href="contact.html">Contact</a></li>
				</ul>
			</nav>

			<div class="colorlib-footer">
				<p><!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
			  Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart" aria-hidden="true"></i> by <a href="https://colorlib.com/" target="_blank">Colorlib</a>
				<ul>
					<li><a href="#"><i class="icon-facebook"></i></a></li>
					<li><a href="#"><i class="icon-twitter"></i></a></li>
					<li><a href="#"><i class="icon-instagram"></i></a></li>
					<li><a href="#"><i class="icon-linkedin"></i></a></li>
				</ul>
			</div>
		</aside> <!-- END COLORLIB-ASIDE -->
		<div id="colorlib-main">
			<section class="ftco-section contact-section">
	      <div class="container">
	        <div class="row d-flex mb-5 contact-info">
	         <div class="col-md-12">
	    						<div class="blog-entry ftco-animate">
	    							
	    							<div class="text pt-2 mt-3">
				              <h3 class="mb-4"><a href="#">Best practices for variant calling using Genome Analysis Toolkit</a></h3>
					          	<span class="category mb-1 d-block"><a href="#">Abstract</a></span>

				              <p class="mb-4">Next generation sequencing (NGS) technologies have significantly expanded clinical genetic 
testing for inherited conditions and diseases like cancer, with accurate variant calling being a 
crucial step that underpins downstream analysis and interpretation. The software tools and 
approaches for detecting sequence variants in clinical samples have evolved significantly, much 
like NGS technologies, in the last decade. Variant calling is a crucial step in analyzing NGS data, 
as it enables the identification of genomic variations such as single nucleotide polymorphisms 
(SNPs), insertions and deletions.</p><p>
The Genome Analysis Toolkit (GATK) is a widely used software suite that provides a range of 
tools for variant calling from NGS data. GATK uses a probabilistic model to identify variants while 
accounting for sequencing errors, mapping errors, and other sources of noise. The GATK workflow 
for variant calling typically involves several preprocessing steps, including mapping the reads to 
a reference genome, marking duplicate reads, and recalibrating the base quality scores. These 
preprocessing steps help improve the accuracy of variant calling.</p><p>
The core variant calling algorithm in GATK is called HaplotypeCaller, which uses local de novo 
assembly to identify haplotypes (i.e., sets of closely linked variants) in the reads. HaplotypeCaller 
then uses these haplotypes to improve the accuracy of variant calling, particularly for regions of 
the genome with complex variants, such as indels. GATK also provides other variant calling tools, 
including UnifiedGenotyper and MuTect, which are optimized for different types of NGS data and 
variant calling applications.</p><p>
After variant calling, the resulting variant call format (VCF) file can be further processed and 
filtered using GATK tools, such as VariantFiltration, to remove low-quality variants and reduce 
false positives. GATK also provides tools for variant annotation, such as VariantAnnotator and 
SnpEff, which can provide additional information about the functional effects of variants, 
including their effects on genes and proteins. Overall, GATK provides a powerful and flexible 
toolkit for variant calling from NGS data. However, effective use of GATK requires expertise in 
bioinformatics and careful attention to quality control and filtering to minimize false positives and 
ensure the accuracy of variant calls. Nonetheless, GATK's popularity and wide adoption in the 
scientific community attest to its usefulness and effectiveness for variant calling from NGS data.
<b>Keywords</b>: GATK, BWA, HaplotypeCaller, SnpEff</p>
 	<span class="category mb-1 d-block"><a href="#">Introduction</a></span>
	<p class="mb-4">Next-generation sequencing (NGS) has revolutionized the field of genomics since its introduction 
over a decade ago. It represents a high-throughput and cost-effective method for sequencing DNA 
and RNA, allowing researchers to decode and analyze genetic information on a scale not 
previously achievable1
. The growth of NGS technology has been remarkable, driving significant 
advancements in various fields and contributing to numerous scientific breakthroughs and medical 
discoveries. NGS has found numerous applications across different areas of research and 
medicine2–6
. In the field of genomics, it has facilitated large-scale projects such as the Human 
Genome Project, which aimed to sequence the entire human genome7,8. NGS has enabled the 
exploration of genomic variations, including single nucleotide polymorphisms (SNPs), insertions, 
deletions, and structural variations, which play critical roles in understanding genetic diseases and 
population genetics.
In the field of clinical diagnostics, NGS has transformed genetic testing by enabling 
comprehensive and rapid analysis of patient samples. It has become a valuable tool for diagnosing 
genetic disorders, identifying disease-causing mutations, and guiding treatment decisions. NGS￾based tests are now routinely used for carrier screening, prenatal testing, newborn screening, and 
oncology diagnostics. Beyond genomics, NGS has extended its applications to other areas of 
research, such as transcriptomics and epigenetics9,10. RNA sequencing (RNA-seq) allows the 
quantification and profiling of gene expression, leading to a deeper understanding of cellular 
functions, developmental processes, and disease mechanisms11–13. Epigenetic modifications, such 
as DNA methylation and histone modifications, can also be analyzed using NGS techniques, 
providing insights into gene regulation and epigenetic alterations associated with diseases14
.
The growth of NGS has been fueled by continuous advancements in sequencing technologies, 
bioinformatics tools, and data analysis methods. The introduction of new platforms and 
improvements in sequencing chemistry and read lengths have enhanced the accuracy, speed, and 
cost-effectiveness of NGS. Moreover, the development of robust bioinformatics pipelines and 
computational algorithms has enabled the efficient processing and interpretation of vast amounts 
of sequencing data.
In this chapter, we discuss the current best practices for variant calling in clinical sequencing 
studies, with a particular emphasis on variant calling for inherited disorders. Variant calling is a 
fundamental step in the analysis of NGS data that involves identifying genetic variations in a 
sample relative to a reference genome. Accurate and reliable variant calling is crucial for 
understanding the genetic basis of diseases, identifying potential drug targets, and developing 
personalized therapies. The Genome Analysis Toolkit (GATK) is a widely used software package 
for variant calling in NGS data. Developed by the Broad Institute, GATK provides a 
comprehensive set of tools for variant discovery, genotyping, and annotation15. The software is 
highly customizable and can be tailored to specific experimental designs and sequencing 
platforms.
<h5>Requirements for running variant calling analysis</h5>
For optimal performance, it is advisable to use a computer with ample memory and processing 
capabilities. While a minimum of 8 GB of memory is necessary, a recommended range of 8 to 16 
GB is suggested when working with larger genomes like the human genome. The software is 
compatible with Unix or Unix-like systems such as Linux or MacOS X, provided Java is installed. 
Although BWA has been adapted for Windows, users have reported utilizing GATK and Picard 
tools with Cygwin; however, this usage is not officially supported. In this protocol we are using 
Ubuntu (Version 22.04) machine for the analysis. The following packages are required to be 
installed in the system to carry out the analysis.</p><p>
Anaconda: Anaconda is a widely used platform in the field of data science. It provides a 
comprehensive and user-friendly environment for data analysis, scientific computing, and machine 
learning. Its ease of use, extensive package library, and robust package management system make 
it a go to choose for many data scientists and researchers. The script to download and install 
anaconda is given below.</p>
<div class="terminal" id="terminal">
  <button class="copy-button" onclick="copyCode()">Copy Code</button>
  <div id="cont">
  <span class="command">wget https://repo.anaconda.com/archive/Anaconda3-2023.03-1-Linux-x86_64.sh</span><br>
  <span class="command">bash Anaconda3-2019.03-Linux-x86_64.sh</span>
</div></div>
Once anaconda is installed, we need to install packages like Burrows-Wheeler Alignment Tool 
(BWA), SA</p>			              
				              
				            </div>
	    						</div>
	    					</div>
	          <div class="w-100"></div>
	          <div class="col-md-3">
	            <p><span>Address:</span> 198 West 21th Street, Suite 721 New York NY 10016</p>
	          </div>
	          <div class="col-md-3">
	            <p><span>Phone:</span> <a href="tel://1234567920">+ 1235 2355 98</a></p>
	          </div>
	          <div class="col-md-3">
	            <p><span>Email:</span> <a href="mailto:info@yoursite.com">info@yoursite.com</a></p>
	          </div>
	          <div class="col-md-3">
	            <p><span>Website</span> <a href="#">yoursite.com</a></p>
	          </div>
	        </div>
	        <div class="row block-9">
	          <div class="col-md-6 order-md-last pr-md-5">
	            <form action="#">
	              <div class="form-group">
	                <input type="text" class="form-control" placeholder="Your Name">
	              </div>
	              <div class="form-group">
	                <input type="text" class="form-control" placeholder="Your Email">
	              </div>
	              <div class="form-group">
	                <input type="text" class="form-control" placeholder="Subject">
	              </div>
	              <div class="form-group">
	                <textarea name="" id="" cols="30" rows="7" class="form-control" placeholder="Message"></textarea>
	              </div>
	              <div class="form-group">
	                <input type="submit" value="Send Message" class="btn btn-primary py-3 px-5">
	              </div>
	            </form>
	          
	          </div>

	          <div class="col-md-6">
	          	<div id="map"></div>
	          </div>
	        </div>
	      </div>
	    </section>

	    <footer class="ftco-footer ftco-bg-dark ftco-section">
	      <div class="container px-md-5">
	        <div class="row mb-5">
	          <div class="col-md">
	            <div class="ftco-footer-widget mb-4 ml-md-4">
	              <h2 class="ftco-heading-2">Category</h2>
	              <ul class="list-unstyled categories">
	                <li><a href="#">Photography <span>(6)</span></a></li>
	                <li><a href="#">Fashion <span>(8)</span></a></li>
	                <li><a href="#">Technology <span>(2)</span></a></li>
	                <li><a href="#">Travel <span>(2)</span></a></li>
	              </ul>
	            </div>
	          </div>
	          <div class="col-md">
	             <div class="ftco-footer-widget mb-4">
	              <h2 class="ftco-heading-2">Archives</h2>
	              <ul class="list-unstyled categories">
	              	<li><a href="#">October 2018 <span>(6)</span></a></li>
	                <li><a href="#">September 2018 <span>(6)</span></a></li>
	                <li><a href="#">August 2018 <span>(8)</span></a></li>
	                <li><a href="#">July 2018 <span>(2)</span></a></li>
	                <li><a href="#">June 2018 <span>(7)</span></a></li>
	              </ul>
	            </div>
	          </div>
	          <div class="col-md">
	            <div class="ftco-footer-widget mb-4">
	            	<h2 class="ftco-heading-2">Have a Questions?</h2>
	            	<div class="block-23 mb-3">
		              <ul>
		                <li><span class="icon icon-map-marker"></span><span class="text">203 Fake St. Mountain View, San Francisco, California, USA</span></li>
		                <li><a href="#"><span class="icon icon-phone"></span><span class="text">+2 392 3929 210</span></a></li>
		                <li><a href="#"><span class="icon icon-envelope"></span><span class="text">info@yourdomain.com</span></a></li>
		              </ul>
		            </div>
	            </div>
	          </div>
	        </div>
	        <div class="row">
	          <div class="col-md-12">

	            <p><!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
	  Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart" aria-hidden="true"></i> by <a href="https://colorlib.com/" target="_blank">Colorlib</a>
	  <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. --></p>
	          </div>
	        </div>
	      </div>
	    </footer>
		</div><!-- END COLORLIB-MAIN -->
	</div><!-- END COLORLIB-PAGE -->

  <!-- loader -->
  <div id="ftco-loader" class="show fullscreen"><svg class="circular" width="48px" height="48px"><circle class="path-bg" cx="24" cy="24" r="22" fill="none" stroke-width="4" stroke="#eeeeee"/><circle class="path" cx="24" cy="24" r="22" fill="none" stroke-width="4" stroke-miterlimit="10" stroke="#F96D00"/></svg></div>


  <script src="js/jquery.min.js"></script>
  <script src="js/jquery-migrate-3.0.1.min.js"></script>
  <script src="js/popper.min.js"></script>
  <script src="js/bootstrap.min.js"></script>
  <script src="js/jquery.easing.1.3.js"></script>
  <script src="js/jquery.waypoints.min.js"></script>
  <script src="js/jquery.stellar.min.js"></script>
  <script src="js/owl.carousel.min.js"></script>
  <script src="js/jquery.magnific-popup.min.js"></script>
  <script src="js/aos.js"></script>
  <script src="js/jquery.animateNumber.min.js"></script>
  <script src="js/bootstrap-datepicker.js"></script>
  <script src="js/jquery.timepicker.min.html"></script>
  <script src="js/scrollax.min.js"></script>
  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBVWaKrjvy3MaE7SQ74_uJiULgl1JY0H2s&amp;sensor=false"></script>
  <script src="js/google-map.js"></script>
  <script src="js/main.js"></script>
    
  </body>

<script>
  function copyCode() {
    const terminal = document.getElementById('cont');
    const code = terminal.innerText;
    navigator.clipboard.writeText(code).then(() => {
     $(".copy-button").html("Copied")
    }).catch(err => {
      console.error("Failed to copy code: ", err);
    });
  }
</script>
</html>